fastify.d.ts

```diff
@@ -7,6 +7,8 @@
 import * as http from 'http'
 import * as http2 from 'http2'
 import * as https from 'https'
+import * as pino from 'pino'
```

Comment:
We decided not to ship pino types. Pino does not have official types support and it relies on `@types/pino`.

Reply:
Yeah the import for it is the same. I downloaded @types/pino as devDep

Reply:
This was done on purpose and should be reverted. For Fastify v2 we decided not to ship these types.